src/CopyrighFixer/CopyrighFixer/signer.cpp

```diff
@@ -7,13 +7,55 @@
 
 #include "signer.h"
 #include "CopyrighFixer/ifilemanager.h"
+#include <QDir>
 
 namespace CopyrighFixer {
 Signer::Signer() {
 
 }
 
-bool Signer::checkSign(const Config &objConf) {
+bool Signer::checkSign(const Config &objConf) {    
+
+    Config currConfig = objConf;
```

Comment:
i think you need to create a separate recursive method with input current dir path argument .

Reply:
You nothing changed on this issue .